### PR TITLE
Issue 74

### PR DESCRIPTION
**Pull request by ongardie**
I think this one is ready to merge into master. @nhardt , want to review?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,15 @@ debug
 logcabin.conf
 smoketest.conf
 smoketeststorage
+
+# Used in building RPM files
+# (I don't know how to get scons to do this underneath /build/)
```
Comment (nhardt) [inline .gitignore]: should these maybe get cleaned up after the build by default?

Comment (nhardt) [inline .gitignore]: also maybe move the .rpm file to build/ ?

**Comment by nhardt** [comment]
looks good overall, will help me out. up to you on those cleanup suggestions.

**Comment by ongardie** [comment]
Yeah, those are good ideas, though I don't know how to make scons do them. Let me try...

**Comment by ongardie** [comment]
I think that does it. Will merge once Travis CI is done

**Pull request merged**